src/PointwiseFunctions/AnalyticSolutions/EinsteinSolutions/KerrSchild.hpp

```diff
@@ -0,0 +1,109 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Options/Options.hpp"
+#include "Utilities/TMPL.hpp"
+
+class DataVector;
+
+namespace EinsteinSolutions {
+
+/*!
+ * \ingroup EinsteinSolutions
+ * \brief Kerr black hole in Kerr-Schild coordinates
+ *
+ * \details
+ * The metric is \f$g_{\mu,\nu} = \eta_{\mu\nu} + 2 H l_\mu l_\nu\f$,
+ * where \f$\eta_{\mu\nu}\f$ is the Minkowski metric, \f$H\f$ is a scalar
+ * function, and \f$l_\mu\f$ is the outgoing null vector.
+ * \f$H\f$ and \f$l_\mu\f$ are known functions of the coordinates
+ * and of the mass and spin vector.
+ *
+ * The following are input file options that can be specified:
+ * Mass (default: 1.), Center (default: {0,0,0}), Spin (default: {0,0,0})
+ */
+class KerrSchild {
+ public:
+  struct Mass {
+    using type = double;
+    static constexpr OptionString help = {"Mass of the black hole"};
+    static type default_value() { return 1.; }
+    static type lower_bounds() { return 0.; }
+  };
+  struct Spin {
+    using type = std::array<double, 3>;
+    static constexpr OptionString help = {"The [x,y,z] spin of the black hole"};
+    static type default_value() { return {{0., 0., 0.}}; }
+    static type lower_bounds() { return {{-1., -1., -1.}}; }
+    static type upper_bounds() { return {{1., 1., 1.}}; }
```

Comment:
I think these do a lexicographic comparison.  Since you have to check the magnitude anyway in the constructor I'd just drop the bounds here.